.github/workflows/main.yml

```diff
@@ -17,9 +17,9 @@ jobs:
         with:
           python-version: 3.9
 
-      - name: Install Twine
+      - name: Install Dependencies
         run: |
-          pip install twine
+          pip install twine click
```

Comment:
Should we also add `click` to the `environment.yaml`?

Reply:
yes, thanks for catching that!